Sources/EnhancedMediaPlayer/SettingsPreference.swift

```diff
@@ -4,7 +4,7 @@ import Combine
 public class SettingsPreferences: ObservableObject {
     public init() {}
     
-    @Published public var isShowingSettingsMenu: Bool = true
+    @Published public var isShowingSettingsMenu: Bool = false
```

Comment:
👀

Reply:
I changed in the last PR, not on purpose